modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+
+[[performance-charts]]
+== Performance
+
+The following charts appear on the *Performance* dashboard:
+
+Inbound - Average Response Time::
+The average response time of inbound requests, grouped in automatic intervals.
+
+Inbound - Average Response Time Grouped by Endpoint::
+The average, maximum, and minimum response times, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Mule Messages::
+The number of Mule Messages processed in the runtime, grouped in automatic intervals.
+
+Inbound Response Time graphs (99, 90, 75, and 50 Percentile)::
+The response time of the 99th, 90th, 75th, and 50th percentile of inbound requests, grouped by application in automatic intervals.
+
+Outbound - Average Response Time::
+The average response time of outbound requests, grouped in automatic intervals.
+
+Outbound - Average Response Time Grouped by HTTP Endpoint::
+The average, maximum, and minimum response times, grouped by endpoint. Cells will vary in color according to the following thresholds:
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Outbound Response Time graphs (99, 90, 75, and 50 Percentile)::
+The response time of the 99th, 90th, 75th, and 50th percentile of outbound requests, grouped in automatic intervals.
+
+[[failures-charts]]
+== Failures
+
+The following charts appear on the *Failures* dashboard:
+
+Total Failed Inbound Requests::
+The sum of failed inbound requests in the period, grouped in automatic intervals.
+
+Total Failed Outbound Requests::
+The sum of failed outbound requests in the period, grouped in automatic intervals.
+
+Inbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
+
+Outbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response and automatic interval.
```

Comment:
"...and automatic interval" makes more sense to me than "in automatic intervals." However, I still think we should clarify what we mean by that: what are the intervals and who/what determines them"

Reply:
@jiang-wu What exactly does automatic intervals refer to here, and how can I standardize this?